dev/app-shell/elements/arc-host.html

```diff
@@ -43,6 +43,7 @@
     }
     async _applyConfig(config) {
       let arc = await this._createArc(config);
+      arc.makeSuggestions = async () => { this._schedulePlanning(state); }
```

Comment:
Is there a call to this somewhere?

Reply:
yep, it is called in the slot-composer ui event handler:
https://github.com/PolymerLabs/arcs/blob/6c62c3ecc4ee9afe3a92d560abc573c99c653640/runtime/slot-composer.js#L181